package.json

```diff
@@ -4,6 +4,9 @@
   "description": "Argus",
   "main": "app.js",
   "scripts": {
+    "start:build": "tsc -w",
+    "start:run": "nodemon build/app.js",
+    "start": "concurrently npm:start:*",
     "test": "mocha **test/*.js --exit"
```

Comment:
Are we using `mocha` for testing? Didn't see any **dev-dependency** though! I'll vote for `Jest` for our testing library; let's discuss about this @zhfahan77 @arafat-al-mahmud